holmes/plugins/toolsets/opensearch/opensearch_query_assist.py

```
@@ -0,0 +1,86 @@
+                type="object",
+                properties={
+                    "id": ToolParameter(type="string", required=True),
+                    "content": ToolParameter(type="string", required=True),
+                    "status": ToolParameter(type="string", required=True),
+                },
+            ),
+        ),
+    }
+
+    def _invoke(
+            self, params: dict, user_approved: bool = False
```

Ruff (0.13.3)

Unused method argument: `user_approved`

(ARG002)